mobile-app/lib/ui/views/podcast/widgets/podcast_title_widget.dart

```diff
@@ -185,214 +188,223 @@ class PodcastTileState extends State<PodcastTile> {
-    return widget.playing
-        ? Card(
-            margin: const EdgeInsets.all(8),
-            elevation: 10,
-            shadowColor: Colors.black,
-            color: const Color.fromRGBO(0x1b, 0x1b, 0x32, 1),
-            child: podcastTile(context),
-          )
-        : podcastTile(context);
+  Widget _buildEpisodeTitle() {
+    return Text(
+      widget.episode.title,
+      style: const TextStyle(
+        fontWeight: FontWeight.bold,
+        fontSize: 20,
+        color: FccColors.gray00,
+      ),
+    );
   }
 
-  ListTile podcastTile(BuildContext context) {
-    return ListTile(
-      title: Row(
-        children: [
-          Expanded(
-            child: Padding(
-              padding: const EdgeInsets.all(8.0),
-              child: Text(
-                widget.episode.title,
-                style: const TextStyle(fontWeight: FontWeight.w700),
-              ),
-            ),
-          ),
-        ],
-      ),
-      onTap: () {
-        Navigator.push(
-          context,
-          MaterialPageRoute(
-            builder: (context) => EpisodeView(
-              episode: widget.episode,
-              podcast: widget.podcast,
-            ),
-            settings: RouteSettings(
-              name: '/podcasts-episode/${widget.episode.title}',
-            ),
+  Widget _buildEpisodeDescription() {
+    final firstSentence = getFirstSentenceFromHtml(widget.episode.description);
+    if (firstSentence.isEmpty) return const SizedBox(height: 14);
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const SizedBox(height: 6),
+        Text(
+          firstSentence,
+          style: const TextStyle(
+            color: FccColors.gray10,
+            fontSize: 17,
           ),
-        );
-      },
-      minVerticalPadding: 16,
-      isThreeLine: true,
-      subtitle: Column(
-        children: [
-          descriptionWidget(),
-          footerWidget(context),
-        ],
+        ),
+        const SizedBox(height: 14),
+      ],
+    );
+  }
+
+  Widget _buildEpisodeDate() {
+    final date = widget.episode.publicationDate;
+    return Text(
+      date != null ? Jiffy.parseFromDateTime(date).fromNow().toUpperCase() : '',
+      style: const TextStyle(
+        fontSize: 13,
+        color: FccColors.gray10,
+        fontWeight: FontWeight.w600,
+        letterSpacing: 1.1,
       ),
     );
   }
 
-  Row footerWidget(BuildContext context) {
-    return Row(
-      children: [
-        (widget._audioService.playbackState.value.processingState ==
-                        AudioProcessingState.loading ||
-                    widget._audioService.playbackState.value.processingState ==
-                        AudioProcessingState.buffering) &&
-                widget._audioService.episodeId == widget.episode.id
-            ? Padding(
-                padding: const EdgeInsets.all(8.0),
+  Widget _buildPlayButton(bool isPlaying) {
+    return Container(
```

Comment:
No need of an extra container. The stylings can be set directly on the button styles itself